champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/passives/Longshot.java

```diff
@@ -141,10 +143,11 @@ public SkillType getType() {
 
     @Override
     public void loadSkillConfig(){
-        baseMaxDamage = getConfig("baseMaxDamage", 12.0, Double.class);
-        maxDamageIncreasePerLevel = getConfig("maxDamageIncreasePerLevel", 1.0, Double.class);
-        baseDamage = getConfig("baseDamage", 0.25, Double.class);
+        baseMaxDamage = getConfig("baseMaxDamage", 14.0, Double.class);
+        maxDamageIncreasePerLevel = getConfig("maxDamageIncreasePerLevel", 2.0, Double.class);
+        baseDamage = getConfig("baseDamage", 0.4, Double.class);
```

Comment:
couldnt minDamage just be baseDamage

Reply:
it could, but I'd prefer a separately customizable value that represents the starting damage for anything future that happens with the skill. Could set it to 0 to have the same effect

Reply:
but if the base damage is 0.4 and the min damage is 1, then the base damage is essentially 1

Reply:
Again, could just make it use baseDamage but being able to customize the minimum damage is nice